packages/realm/src/OrderedCollection.ts

```diff
@@ -178,6 +178,12 @@ export abstract class OrderedCollection<T = unknown, EntryType extends [unknown,
       writable: false,
       value: mixedToBinding.bind(undefined, realm.internal),
     });
+    // See https://tc39.es/ecma262/multipage/fundamental-objects.html#sec-symbol.unscopables
+    Object.defineProperty(this, Symbol.unscopables, {
+      enumerable: false,
+      configurable: true,
```

Comment:
I am looking forward to understand the implication of `true` here

Reply:
Yeah I'd like to know that too 😉 Good that you commented, I noticed the link was wrong so now I've updated that: https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype-@@unscopables